common/src/main/java/com/trainlab/repository/UserRepository.java

```diff
@@ -6,10 +6,9 @@
 import java.util.List;
 import java.util.Optional;
 
-//@Cacheable("users")
 public interface UserRepository extends JpaRepository<User, Long> {
     Optional<User> findByAuthenticationInfoEmail(String email);
 
     List<User> findAllByIsDeletedFalseOrderById();
-
+    User findByIsDeletedFalseAndId(Long id);
```

Comment:
Обычно аргументы с дефолтным значением описываются в конце
И желательно все методы на поиск сущности из БД делать Optional чтобы избежать NPE

```suggestion
    Optional<User> findByIdAndIsDeletedFalse(Long id);
```